### PR TITLE
Add Jina (Small) on Garden signal

### PR DESCRIPTION
**Pull request by dsmilkov**

### CODE REVIEW

File: lilac/embeddings/jina_garden.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+from ..schema import Item, lilac_embedding
+from ..signal import TextEmbeddingSignal
+from ..tasks import TaskExecutionType
+
+JINA_CONTEXT_SIZE = 8192
+
+
+class JinaV2SmallGarden(TextEmbeddingSignal):
+  """Jina V2 Embeddings with 8K context, hosted on Garden.
```
Comment (nsthorat) [inline lilac/embeddings/jina_garden.py:19]: maybe we should use "Lilac Garden" when we say Garden

File: lilac/embeddings/jina_garden.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+from lilac.batch_utils import compress_docs
+
+from ..schema import Item, lilac_embedding
+from ..signal import TextEmbeddingSignal
+from ..tasks import TaskExecutionType
+
+JINA_CONTEXT_SIZE = 8192
+
+
+class JinaV2SmallGarden(TextEmbeddingSignal):
```
Comment (nsthorat) [inline lilac/embeddings/jina_garden.py:18]: in the background we should think about whether garden should just become a bit to the embedding (it's not too important if you compute this on garden or locally, as long as the outputs are the same)

**Review by nsthorat — APPROVED** [review]

File: lilac/embeddings/jina_garden.py (hunk 1 above)
Comment (dsmilkov) [reply lilac/embeddings/jina_garden.py:19]: done

File: lilac/embeddings/jina_garden.py (hunk 2 above)
Comment (dsmilkov) [reply lilac/embeddings/jina_garden.py:18]: Yeah, i like that. we can re-work this later easily... we can also add that bit to the dataset.cluster

**Pull request merged**